src/main/java/frc/robot/RobotContainer.java

```diff
@@ -81,18 +81,19 @@ public RobotContainer(RobotId robotId) {
         index = new StorageIndex();
         break;
       case Vertigo:
-          drive = new Drivetrain(vision);
-          driver = new GameControllerDriverConfig(drive, arm, index, shooter);
-          driver.configureControls();
-          drive.setDefaultCommand(new DefaultDriveCommand(drive, driver));
-          break;
+        drive = new Drivetrain(vision);
+        driver = new GameControllerDriverConfig(drive, arm, index, shooter);
+        driver.configureControls();
+        drive.setDefaultCommand(new DefaultDriveCommand(drive, driver));
+        break;
         
       default:
       case SwerveCompetition:
         // arm = new Arm();
         intake = new Intake();
         index = new StorageIndex();
         shooter = new Shooter();
+        registerCommands();
```

Comment:
put a break

Reply:
A break would cause SwerveCompetition to not have a drivetrain.

Reply:
Oh, okay, I see that now. Thanks for clarifying, yeah, I guess it needs to continue running the next case code